src/components/DepositWidget/Styled.tsx

```diff
@@ -95,27 +95,19 @@ export const RowClaimButton = styled.button`
   }
 `
 
-export const RowClaimLink = styled.a`
-  text-decoration: none;
-  cursor: pointer;
+export const RowClaimSpan = styled.span`
```

Comment:
why not make this a `div` if using display block? (curious)

Reply:
div is generally not to be used inside button
only phrasing content
or so says the spec
for example `<button><div/></button>` would fail w3c validation, not that really matters

Reply:
But you are right that it doesn't have to be `block`, anything inside a flex is already a `block`. Will remove